conda/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set data= load_setup_py_data() %}
+package: 
+  name: ts-xml
+  version: {{ data.get('version') }}
+
+source:
+  path: ..
+
+build:
+  noarch: generic
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+test:
+  requires:
+    - astropy
+    - lxml
+  source_files:
+    - python
+    - tests
+    - setup.cfg
+    - pyproject.toml
+  commands:
+    - pytest
+
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+    - setuptools_scm
+    - setuptools
+  build:
+    - python {{ python }}
+    - setuptools_scm
+    - setuptools
+  run:
+    - python {{ python }}
```

Comment:
I think you also need `astropy` and `lxml` as run dependencies (but no `ts-conda-build`, which is only a test dependency).